lib/buckets_mdapi_client.js

```diff
@@ -41,38 +40,29 @@ function createClient(options, callback) {
 
         log.info({
             url: pnodeUrl
-        }, 'creating boray client');
+        }, 'creating buckets-mdapi client');
 
-        var borayargs = clone(options.borayOptions);
-        if (!borayargs.cueballOptions) {
-            borayargs.cueballOptions = {};
+        var buckets_mdapi_args = clone(options.bucketsMdapiOptions);
+        if (!buckets_mdapi_args.cueballOptions) {
+            buckets_mdapi_args.cueballOptions = {};
         }
-        borayargs.unwrapErrors = true;
-        borayargs.srvDomain = pnodeUrl.hostname;
-        borayargs.cueballOptions.defaultPort = parseInt(pnodeUrl.port, 10);
-        borayargs.log = options.log.child({
-            component: 'BorayClient',
+        buckets_mdapi_args.unwrapErrors = true;
+        buckets_mdapi_args.srvDomain = pnodeUrl.hostname;
+        buckets_mdapi_args.cueballOptions.defaultPort =
+            parseInt(pnodeUrl.port, 10);
+        buckets_mdapi_args.log = options.log.child({
+            component: 'BucketsMdapiClient',
             pnode: pnodeUrl.hostname
         });
-        borayargs.crc_mode = options.crc_mode;
+        buckets_mdapi_args.crc_mode = options.crc_mode;
 
-        var client = boray.createClient(borayargs);
+        var client = buckets_mdapi.createClient(buckets_mdapi_args);
         clientMap[pnode] = client;
         clientArray.push(client);
-
-        if (clientArray.length === pnodes.length) {
-            // write ready cookie when clients have connected
-            log.info('all boray clients instantiated writing ready cookie');
-            try {
-                fs.writeFileSync('/var/tmp/electric-boray-ready', null);
-            } catch (e) {
-                throw new verror.VError(e, 'unable to write ready cookie');
-            }
-        }
```

Comment:
This was cruft?

Reply:
Yeah, that was carryover from when this module lived in `electric-boray`/`buckets-mdplacement`.